fili-generic-example/src/main/java/com/yahoo/wiki/webservice/data/config/auto/ConfigLoader.java

```diff
@@ -0,0 +1,12 @@
+// Copyright 2016 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.wiki.webservice.data.config.auto;
+
+import java.util.List;
+
+/**
+ * Created by kevin on 3/3/2017.
+ */
+public interface ConfigLoader {
+    public List<? extends DruidConfig> getTableNames();
```

Comment:
Ok, in general comments on classes are a good idea, and necessary on interfaces.

However, I'm not sure I understand the connection between this interface and it's name.  Further, there's already a class ConfigurationLoader, which I think this interface could easily be confused with.

Also, this interface appears to be a single method function.  It's possible rather than have this be an explicit interface, you could instead make anything that is currently implementing it instead to implement:

Supplier<List<? extends DruidConfig>>

https://docs.oracle.com/javase/8/docs/api/java/util/function/Supplier.html